src/ralph/accounts/templates/ralphuser/user_additional_info.html

```diff
@@ -0,0 +1,14 @@
+{% extends BASE_TEMPLATE %}
+{% load i18n table %}
+{% block view_content %}
+<div class="large-12 columns">
+  <h1>{{ object.first_name }} {{ object.last_name }}</h1>
+  <br /> <br />
+  <h5>{% trans 'Assigned assets' %}:</h5>
+
+  {% table asset_list 'id' 'model__category' 'model__manufacturer' 'model__name' 'sn' 'barcode' 'remarks' 'status' %}
```

Comment:
imo it'd be better to define this list in python (somewhere in `RalphDetailView`). @ar4s what do you think?